components/health_controller/src/types.rs

```diff
@@ -84,36 +92,40 @@ impl Debug for LatencyInspector {
 }
 
 impl LatencyInspector {
-    pub fn new(id: u64, cb: Box<dyn FnOnce(u64, RaftstoreDuration) + Send>) -> Self {
+    pub fn new(id: u64, cb: Box<dyn FnOnce(u64, UnifiedDuration) + Send>) -> Self {
         Self {
             id,
             cb,
-            duration: RaftstoreDuration::default(),
+            duration: UnifiedDuration::default(),
         }
     }
 
     pub fn record_store_wait(&mut self, duration: std::time::Duration) {
-        self.duration.store_wait_duration = Some(duration);
+        self.duration.raftstore_duration.store_wait_duration = Some(duration);
     }
 
     pub fn record_store_process(&mut self, duration: std::time::Duration) {
-        self.duration.store_process_duration = Some(duration);
+        self.duration.raftstore_duration.store_process_duration = Some(duration);
     }
 
     pub fn record_store_write(&mut self, duration: std::time::Duration) {
-        self.duration.store_write_duration = Some(duration);
+        self.duration.raftstore_duration.store_write_duration = Some(duration);
     }
 
     pub fn record_store_commit(&mut self, duration: std::time::Duration) {
-        self.duration.store_commit_duration = Some(duration);
+        self.duration.raftstore_duration.store_commit_duration = Some(duration);
     }
 
     pub fn record_apply_wait(&mut self, duration: std::time::Duration) {
-        self.duration.apply_wait_duration = Some(duration);
+        self.duration.raftstore_duration.apply_wait_duration = Some(duration);
     }
 
     pub fn record_apply_process(&mut self, duration: std::time::Duration) {
-        self.duration.apply_process_duration = Some(duration);
+        self.duration.raftstore_duration.apply_process_duration = Some(duration);
+    }
+
+    pub fn record_network(&mut self, duration: std::time::Duration) {
```

Comment:
How about
```suggestion
    pub fn record_network_transmit(&mut self, duration: std::time::Duration) {
```
or `record_network_io_duration(...)`